app/controllers/comments_controller.rb

```diff
@@ -1,10 +1,39 @@
 class CommentsController < ApplicationController
+  def new
+    @article = Article.find(params[:id])
```

Comment:
Note: now in `CommentsController` the id that is mean to `article` is `article_id`
You can see from `rails routes | grep comment`